media-libs/libhdhomerun/libhdhomerun-20190621.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+DESCRIPTION="SiliconDust HDHomeRun Utilties"
+HOMEPAGE="https://www.silicondust.com/support/linux/"
+SRC_URI="https://download.silicondust.com/hdhomerun/${PN}_${PV}.tgz"
+
+PATCHES=(
+	"${FILESDIR}"/dont-strip.patch
+)
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${PN}"
+
+src_prepare() {
+	default
+}
+
+src_configure() {
```

Comment:
Why do you need that? A comment in the ebuild would be helpful.

Reply:
Remove not needed